README.md

```diff
@@ -11,6 +11,7 @@
 
 - HTML
 - CSS
+- JavaScript
 
 ## Live Demo (if available)
 
```

Comment:
- [ ] [OPTIONAL] Please, make sure to remove the  "if available" part from the "Live Demo" Section